### PR TITLE
docs/podman*.md: fix numerous option typos and spacing errors

### PR DESCRIPTION
**Pull request by rpjday**
Cursory examination of man pages shows a number of typos:

  - missing hyphens
  - missing blank line
  - longer option should precede shorter option

This is not an extensive fix, there's still a lot that could
be cleaned up.

Signed-off-by: Robert P. J. Day <rpjday@crashcourse.ca>

### CODE REVIEW

**Comment by rhatdan** [comment]
@rpjday Awesome work
LGTM
@TomSweeneyRedHat @mheon @baude @giuseppe @vrothberg PTAL

File: docs/podman-volume-inspect.1.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ existing volumes, use the **--all** flag.
 
 ## OPTIONS
 
-**-a**, **--all**=""
+**-a**, **--all**
```
Comment (TomSweeneyRedHat) [inline docs/podman-volume-inspect.1.md]: missed one flip it looks like.

**Comment by TomSweeneyRedHat** [comment]
I know this is merged, but just wanted to say thanks @rpjday !  Nice catches/fixes.

**Pull request merged**